app/assets/stylesheets/application.scss

```
@@ -27,6 +27,12 @@
   }
 }
 
+.govuk-width-container {
+  max-width: 1140px;
+  padding-left: 15px;
+  padding-right: 15px;
+}
```

```suggestion
.govuk-width-container--large {
  max-width: 1140px;
  padding: 0 govuk-spacing(3);
}
```

I would add it as an additional new BEM modifier rather than overriding the original styling. 

Also do we need the padding? should it not just keep the padding that exists?

The implementation will be as follows:

```html
<div class="govuk-width-container govuk-width-container--large"></div>
```

---

cc @kevindew - do you have anything to add to this?

---

Also, 1140px is a very odd number (how did we get to this), does 1200px not work?

---

I wasn't in the meeting but i think 1140px was decided on as it's what Boostrap has. And the idea was it would be less jarring when users jump from page to page between the design systems.